src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -680,7 +680,9 @@ bool GcInfoDecoder::EnumerateLiveSlots(
     // previously visited a child funclet
     if (WantsReportOnlyLeaf() && (inputFlags & ParentOfFuncletStackFrame))
     {
-        LOG((LF_GCROOTS, LL_INFO100000, "Not reporting this frame because it was already reported via another funclet.\n"));
+#ifndef NATIVEAOT        
+        STRESS_LOG0(LF_GCROOTS, LL_INFO100, "Not reporting this frame because it was already reported via another funclet.\n");
```

Comment:
I think that `LL_INFO100` too verbose log level for GC roots reporting.

Reply:
Reverted this change as it was testing only